happy/utils/IP.py

```diff
@@ -111,8 +111,10 @@ def prefixMatchAddress(prefix, addr):
         if prefix is None:
             return False
 
+        if prefix not in addr:
```

Comment:
I will remove the change in IP.py because I found root cause for the issue in another place.
I will open a separate PR for IP.py fix.

Reply:
I have removed change in IP.py, because https://github.com/openweave/happy/pull/38 has the right fix.